pkg/proc/native/registers_linux_loong64.go

```diff
@@ -0,0 +1,142 @@
+//go:build linux && loong64
+// +build linux,loong64
+
+package native
+
+import (
+	"debug/elf"
+	"encoding/binary"
+	"syscall"
+	"unsafe"
+
+	sys "golang.org/x/sys/unix"
+
+	"github.com/go-delve/delve/pkg/dwarf/op"
+	"github.com/go-delve/delve/pkg/proc"
+	"github.com/go-delve/delve/pkg/proc/linutil"
+)
+
+const (
+	// Refer to the definition of struct user_pt_regs in the kernel file ptrace.h
+	_LOONG64_GREGS_SIZE = (32 * 8) + 8 + 8 + 8 + (10 * 8)
+
+	// In fact, the total number of bytes is 268(32 fpr * 8 + 1 Fcc * 8 + 1 Fcsr * 4),
+	// but since the Len defined in sys.Iovec is uint64,the total numbel of bytes must
+	// be an integral multiple of 8,so add 4 bytes
+	_LOONG64_FPREGS_SIZE = (32 * 8) + (1 * 8) + (1 * 4) + 4
+)
+
+func ptraceGetGRegs(pid int, regs *linutil.LOONG64PtraceRegs) (err error) {
+	iov := sys.Iovec{Base: (*byte)(unsafe.Pointer(regs)), Len: _LOONG64_GREGS_SIZE}
```

Comment:
Why not use `unsafe.Sizeof` instead of a hardcoded constant?

Reply:
I use the same coding style as arm64 and ppc64le.